react/components/admin/redirects/UploadModal/UploadPrompt.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { useState } from 'react'
+import ReactDropzone, { ImageFile } from 'react-dropzone'
+import { FormattedMessage } from 'react-intl'
+import { Radio } from 'vtex.styleguide'
+
+import PaperIcon from '../../../icons/PaperIcon'
+import { UploadActionType } from '../mutations/SaveRedirectFromFile'
+
+interface Props {
+  hasRedirects: boolean
+  saveRedirectFromFile: (
+    acceptedFiles: ImageFile[],
+    uploadActionType: UploadActionType
+  ) => void
+}
+
+const UploadPrompt: React.FC<Props> = ({
+  hasRedirects,
+  saveRedirectFromFile,
+}) => {
+  const [uploadActionType, setUploadActionType] = useState<UploadActionType>(
+    'merge'
+  )
+
+  return (
+    <>
+      <h1 className="self-start fw3 mv0">
+        <FormattedMessage id="pages.admin.redirects.upload-modal.prompt.title" />
+      </h1>
+      <p className="f6 lh-copy">
+        <FormattedMessage id="pages.admin.redirects.upload-modal.prompt.body.first" />
+        <a
+          className="c-action-primary link hover-c-action-primary pointer"
+          download
+          href="/_v/private/pages/redirects_template.csv"
+        >
+          <FormattedMessage id="pages.admin.redirects.upload-modal.prompt.body.sample-download" />
+        </a>
+        {hasRedirects && (
+          <>
+            {' '}
```

Comment:
:pick: There are quite a few `{' '}`; are they necessary?

Reply:
Yes, otherwise there's no space on the text.

Reply:
Got it, but perhaps white spaces could be added to the i18n texts.